include/openthread/platform/toolchain.h

```diff
@@ -103,6 +118,8 @@ extern "C" {
 #define OT_TOOL_PACKED_FIELD __attribute__((packed))
 #define OT_TOOL_PACKED_END __attribute__((packed))
 #define OT_TOOL_WEAK __attribute__((weak))
+#define OT_TOOL_ALIGNED_FIELD_BEGIN(size)
```

Comment:
Given that these macros are no longer needed in this PR, can we avoid making changes to this file in this PR?

Reply:
I've removed the commit with the uneccessary macros.